snapcraft.yaml

```diff
@@ -103,9 +103,13 @@ parts:
 #     lower-than: '49'
     after: [ gedit-gtksourceview, amtk, tepl ]
     source: https://gitlab.gnome.org/World/gedit/gedit.git
-    source-tag: '48.2'
+    source-tag: '48.1'
     source-depth: 1
     source-type: git
+# ext:updatesnap
```

Comment:
This is duplicating the existing `updatesnap` snippet instead of updating it

Reply:
Bad rebase! Thanks.